lib/shopify-cli/commands/create.rb

```diff
@@ -32,10 +32,10 @@ def self.all_visible_type
       end
 
       def self.help
-        project_types = all_visible_type.map(&:project_type).join("|")
+        project_types = all_visible_type.map(&:project_type).join(" | ")
```

Comment:
This change here was just an inconsistency in the help text that I noticed and slipped it into this PR